plugins/user-settings/src/components/General/UserSettingsMenu.tsx

```diff
@@ -48,7 +53,12 @@ export const UserSettingsMenu = () => {
         <MoreVertIcon />
       </IconButton>
       <Menu anchorEl={anchorEl} open={open} onClose={handleClose}>
-        <MenuItem data-testid="sign-out" onClick={() => identityApi.signOut()}>
+        <MenuItem
+          data-testid="sign-out"
+          onClick={() =>
+            identityApi.signOut().catch(error => errorApi.post(error))
```

Comment:
Would you be able to update the corresponding test for UserSettingsMenu and assert that we post the error to the errorApi.

An example of another test doing that can be found in `packages/core-components/src/layout/ErrorBoundary/ErrorBoundary.test.tsx`

Reply:
I've added a test, let me know if it's fine as-is.